### DISCUSSION TITLE
Inhalation of a macromolecule in preclinical species

### DISCUSSION THREAD

**Discussion by RobinM92** [Open-Systems-Pharmacology/Forum#920] [Questions & Problems] [answered]
Dear community, 

I am currently working on the design of an inhalation experiment of a new biologic in non-human primates. For this I thought to use the OSP to combine the information I have at hand. I would like to pick your brains in how to combine what is already available in PKSim and MoBi in the most efficient way to predict both lung exposure and systemic exposure after an inhalation. I have IV data, disposition patterns and an idea about the fraction of the administered dose that reaches the respiratory tract. My idea was thus to use one of the two inhalation models which have been constructed in MoBi: https://github.com/Open-Systems-Pharmacology/Inhalation-model or https://github.com/Open-Systems-Pharmacology/Lung-absorption-model 
The problem is of course that these are completely parameterized for humans and not for NHP. Therefore I thought to create a PKSim model first including NHP and the specific characteristics of my biologic, confirm this with the IV data and then export it to MoBi. So far so good, but what would in your opinion be the best way to combine this exported model with the developed inhalation models? Is there a way to scale down spatial structures in these lung models with body weight (which would be a very crude approach but would satisfy my current purposes). 

Thank you very much for your insights!
Kind regards,

Robin

**Comment by moPellowe** [comment, marked as answer]
Dear Robin,

I can speak to the first inhalation model (https://github.com/Open-Systems-Pharmacology/Inhalation-model). 

You can scale down the spatial structures as you described. To do this most efficiently, I would access the current parameter values (e.g. airway diameter, airway length, etc.) and scale them using the R toolbox.

In terms of connecting this inhalation model to a PK-Sim model, the model is not yet configured to do this very easily. The inhalation model needs to be connected to the full body PK-Sim model in a way that allows for intermittent deposits of molecule into the oral pathway due to mucociliary clearance. This configuration step still needs to be done with this inhalation model and it would take some work. I'm not sure if the other inhalation model would be easier to connect to PK-Sim.

Alternatively, you could use the inhalation model connected to a two-compartment model (as it is currently configured), which would give you an estimate of both lung and systemic exposure after inhalation.

Let me know if you have any other questions or concerns about configuring or using the model.

Thank you,
Moriah Pellowe

**Reply by RobinM92** [reply]
Dear Moriah,
Thank you for your quick response! Would you use one of the r-scripts in the github repository for this or the ospsuite itself? As I have some IV data which I fitted to a 2-cmt model, I would say that connecting the lung model to this one would be enough for now. My last concern is the fact that the compound under study is a protein and if I am not mistaken things such as alveolar macrophagal uptake or lung proteases are not included in this (or any) model, and neither are specific protein uptake processes. If I select the compound to be a large molecule all diffusion-related processes are set to 0, so I am wondering how (and if, experimental data coming soon!) the compound enters the plasma compartment and how we would model this.

Thank you very much!
Kind regards,
Robin

**Reply by moPellowe** [reply]
Dear Robin,
You would update the spatial structure parameter values using ospsuite itself.

You are correct in that the model does not take into account alveolar macrophagal uptake, lung proteases, or specific protein uptake processes. With some work, these processes could be incorporated into the inhalation model either as a single clearance rate or as individual processes.

If I remember correctly, the inhalation model does not distinguish between small and large molecules. In other words, I think that large molecules are still subject to diffusion. This can be updated as the model has not yet been used for large molecules. The question of how large molecules enter the plasma compartment (if not by diffusion) is one that I don't know the answer to offhand as I have not worked with inhalation of large molecules before. There may be additional uptake or clearance processes that are needed for large molecules.

Thank you,
Moriah Pellowe
